docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -82,12 +82,12 @@ yarn add @material-ui/styles@next
 
 - ⚠️ Material-UI depends on JSS v10. JSS v10 is not backward compatible with v9.
   Make sure JSS v9 is not installed in your environment.
-  Removing `react-jss` from your package.json can help.
+  Removing `react-jss` from your `package.json` can help.
 - Remove the first option argument of `withTheme()`.
   The first argument was a placeholder for a potential future option.
   We have never found a need for it.
   It's time to remove this argument.
-  It matches the emotion and styled-components API.
+  It matches the [emotion](https://emotion.sh/docs/introduction) and styled-components API.
```

Comment:
Either we link both or none.

Reply:
I appended link for both :)

Reply:
I can't see the link to styled-components. Did you push it?

Reply:
Oops sorry, got lost in merging conflict. Added now ~